compiler/circle-mpqsolver/src/MPQSolver.h

```diff
@@ -25,6 +26,24 @@
 namespace mpqsolver
 {
 
+using LayerParam = luci::CircleQuantizer::Options::LayerParam;
+using LayerParams = std::vector<std::shared_ptr<LayerParam>>;
+
+enum class QuantizationPattern
+{
+  LayerNorm
```

Comment:
Maybe better to use a more descriptive name, e.g., Q8LayerNormWithQ16Variance

Reply:
> Maybe better to use a more descriptive name, e.g., Q8LayerNormWithQ16Variance

Ok.